migrations.lua

```diff
@@ -31,6 +31,7 @@ local types = schema.types
 return {
     -- TODO: We will eventually create migrations for the other tables.
 
+    -- Create Collections and CollectionMemberships
     [20190140] = function()
```

Comment:
WAT. I don't know how migrations work at all. Where does this magic number come from?

Reply:
The key is mostly up to you. They really just need to be unique. Most of the time you use dates because they give some logical ordering to migrations, and nearly always migrations are run in order.

Though that key should end in `040`, I believe. :/ 

I listed the format here: https://github.com/bromagosa/snapCloud/blob/master/migrations.lua#L4
The additional digit is just incase 2 migrations are created on the same day, which could happen at some point. (And > 10 migrations on the same day seems rather unrealistic.) 
Other migrations tools use ISO timestamps to the second (or microsecond) level to avoid any conflicts, but that didn't seem work the hassle. 

I wonder if we should use real strings for keys. (Everything should work fine, and we could actually separate them with dashes (`['2019-01-04-0']`)...)